CONTRIBUTING.md

```diff
@@ -53,12 +53,84 @@ Save it with to the `source/img/commons-logos/ folder, with **compression enable
 ```
 # Using GIMP
 # File > Export As (Shift + Ctrl + E)
-# Choose PNG format, enable "Save backgroung color" and add some compression level
+# Choose PNG format, enable "Save background color" and add some compression level
 ```
 
-Finally, after you've succesfully tested the changes, using our [development guidelines](https://github.com/openshift/commons.openshift.org#development), submit a Pull Request using the guidelines bellow.
+Finally, after you've successfully tested the changes, using our [development guidelines](https://github.com/openshift/commons.openshift.org#development), submit a Pull Request using the guidelines bellow.
```

Comment:
"https://github.com/openshift/commons.openshift.org#development" is now "https://github.com/openshift-cs/commons.openshift.org#development"